easybuild/easyconfigs/g/GroopM/GroopM-0.3.4-foss-2016b-Python-2.7.13.eb

```diff
@@ -0,0 +1,46 @@
+##
+#  This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+#  Homepage: https://www.adelaide.edu.au/phoenix/
+# 
+#  Copyright::   adelaide.edu.au/phoenix
+#  Authors:: Robert Qiao <robert.qiao@adelaide.edu.au>, Exe Escobedo <exequiel.sepulvedaescobedo@adelaide.edu.au>
+#  License:: MIT 
+#
+#  Notes::
+###
+
+easyblock = 'PythonPackage'
+
+name = 'GroopM'
+version = '0.3.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://ecogenomics.github.io/GroopM/'
+description = """ GroopM is a metagenomic binning toolset. It leverages spatio-temoral
+dynamics (differential coverage) to accurately (and almost automatically)
+extract population genomes from multi-sample metagenomic datasets. 
+"""
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('Python', '2.7.13'),
+    ('Cython', '0.25.2', versionsuffix),
+    ('numpy', '1.11.1', versionsuffix),
+    ('scipy', '0.17.1', versionsuffix),
```

Comment:
@boegel This package is compiled against specific numpy and scipy version, any suggestions?